test/support/coverage.js

```diff
@@ -1,28 +1,36 @@
-function writeCoverage(map) {
-  const coverageFile = `${ Cypress.config('coverageFolder') }/out.json`;
+import { each, uniq } from 'underscore';
 
-  return cy.writeFile(coverageFile, map);
+let windowCoverageObjects;
+
+function storeCoverage(win) {
+  const coverage = win.__coverage__;
+
+  if (!coverage) return;
+
+  windowCoverageObjects.push(coverage);
 }
 
 if (Cypress.env('COVERAGE')) {
+  beforeEach(function() {
```

Comment:
In attempting to switch to `@cypress/code-coverage` I found this library does not let us combine coverage reports across tests when just running cypress.  And it was actually slightly slower..  _but_ I copied the pattern of capturing the coverage object on the window load and then consuming in it the after.  This _should_ allow for us to use `cy.visit` without losing coverage.